### ISSUE TITLE
Game crashes whenever I click a link

### ISSUE THREAD

**Issue by fallerOfFalls** [anegostudios/VintageStory-Issues#355]
**Game Version:** 1.11.0
**Platform:** Linux - Debian 9
**Modded:** No - survival

### Description
Whenever I click an external link, the game crashes.

### How to reproduce
1. Click on the "update to latest game version" link, or the "survival starter guide" link.
2. It crashes

### Expected behavior
opens the link in the default web browser

### Logs
There doesn't seem to be a logs folder.

**Comment by copygirl** [comment]
The logs folder should be in the game's data folder, where saves and user mods are stored. You should be able to get to it by going to the *Mod Manager* in the main menu, clicking the button to open the mod folder and going one folder up. For me it's in `~/.config/VintagestoryData/Logs`.

**Comment by fallerOfFalls** [comment]
Thanks!
last entry in client-crash.txt
```Version: v1.11.0 (Stable)28/12/2019 19:53:47: Critical error occurred
System.ComponentModel.Win32Exception (0x80004005): Cannot find the specified file
  at System.Diagnostics.Process.StartWithShellExecuteEx (System.Diagnostics.ProcessStartInfo startInfo) [0x00102] in <dfe42095fd31410e95d8021f755c3557>:0 
  at System.Diagnostics.Process.Start () [0x00032] in <dfe42095fd31410e95d8021f755c3557>:0 
  at (wrapper remoting-invoke-with-check) System.Diagnostics.Process.Start()
  at System.Diagnostics.Process.Start (System.Diagnostics.ProcessStartInfo startInfo) [0x0001b] in <dfe42095fd31410e95d8021f755c3557>:0 
  at System.Diagnostics.Process.Start (System.String fileName) [0x00006] in <dfe42095fd31410e95d8021f755c3557>:0 
  at Vintagestory.API.Client.GuiElementNewVersionText.OnMouseDownOnElement (Vintagestory.API.Client.ICoreClientAPI api, Vintagestory.API.Client.MouseEvent args) [0x00029] in <6b0eaf01de5d44b2b818ce377912bf10>:0 
  at Vintagestory.API.Client.GuiElement.OnMouseDown (Vintagestory.API.Client.ICoreClientAPI api, Vintagestory.API.Client.MouseEvent mouse) [0x00014] in <6b0eaf01de5d44b2b818ce377912bf10>:0 
  at Vintagestory.API.Client.GuiComposer.OnMouseDown (Vintagestory.API.Client.MouseEvent mouseArgs) [0x00022] in <6b0eaf01de5d44b2b818ce377912bf10>:0 
  at _cNt1fiVWhGOpmOGPUmVHVeti6lG._D0AqJwJXrSyq4yH81mBFsrujtcd (Vintagestory.API.Client.MouseEvent ) [0x00056] in <53b5f2473c3848b6a26bc4a7798e9ea5>:0 
  at _XeKQjvZ5CHOpqKVCJia8w1ngAXo._D0AqJwJXrSyq4yH81mBFsrujtcd (Vintagestory.API.Client.MouseEvent ) [0x00008] in <53b5f2473c3848b6a26bc4a7798e9ea5>:0 
  at _Uzs3ZIOcbeuAACiw0HSA5XAvMcKA._D0AqJwJXrSyq4yH81mBFsrujtcd (Vintagestory.API.Client.MouseEvent ) [0x00025] in <53b5f2473c3848b6a26bc4a7798e9ea5>:0 
  at _k7exH1LagwKlon0bbWt5grgXY46._r0iSc8tb9daT0vcZeCYWZW9U5pe (System.Object , OpenTK.Input.MouseButtonEventArgs ) [0x0002c] in <53b5f2473c3848b6a26bc4a7798e9ea5>:0 
  at (wrapper delegate-invoke) System.EventHandler`1[OpenTK.Input.MouseButtonEventArgs].invoke_void_object_TEventArgs(object,OpenTK.Input.MouseButtonEventArgs)
  at OpenTK.NativeWindow.OnMouseDown (OpenTK.Input.MouseButtonEventArgs e) [0x00000] in <d2cd3fbe958341e98eff3ee31a301de2>:0 
  at OpenTK.NativeWindow.OnMouseDownInternal (System.Object sender, OpenTK.Input.MouseButtonEventArgs e) [0x00000] in <d2cd3fbe958341e98eff3ee31a301de2>:0 
  at (wrapper delegate-invoke) System.EventHandler`1[OpenTK.Input.MouseButtonEventArgs].invoke_void_object_TEventArgs(object,OpenTK.Input.MouseButtonEventArgs)
  at OpenTK.Platform.NativeWindowBase.OnMouseDown (OpenTK.Input.MouseButton button) [0x0002e] in <d2cd3fbe958341e98eff3ee31a301de2>:0 
  at OpenTK.Platform.X11.X11GLNative.ProcessEvents () [0x00831] in <d2cd3fbe958341e98eff3ee31a301de2>:0 
  at OpenTK.NativeWindow.ProcessEvents (System.Boolean retainEvents) [0x00035] in <d2cd3fbe958341e98eff3ee31a301de2>:0 
  at OpenTK.NativeWindow.ProcessEvents () [0x00000] in <d2cd3fbe958341e98eff3ee31a301de2>:0 
  at OpenTK.GameWindow.Run (System.Double updates_per_second, System.Double frames_per_second) [0x000da] in <d2cd3fbe958341e98eff3ee31a301de2>:0 
  at OpenTK.GameWindow.Run () [0x00000] in <d2cd3fbe958341e98eff3ee31a301de2>:0 
  at _BpmYDwte9UgHL1NeHQCFwWPUFoAA._Sczbg0FjRjmNsd8yOfTFweX0FGBA (_dQW71ep8GlJRb0IDfmlVgByPa4e , System.String[] ) [0x002f7] in <8b1ddc1eb120400eb6394774e5d4aeb4>:0 
  at _BpmYDwte9UgHL1NeHQCFwWPUFoAA+_txaBjCMDNhHMh3nvUm9Ku9j6cTq._lRtFntg1ukuV3T03AELXrETDhyK () [0x00000] in <8b1ddc1eb120400eb6394774e5d4aeb4>:0 
  at _NMmYWPalISDmLBuhcXuBforA4SJ._Sczbg0FjRjmNsd8yOfTFweX0FGBA (System.Threading.ThreadStart ) [0x00007] in <53b5f2473c3848b6a26bc4a7798e9ea5>:0 
```
Is that what you need?

**Comment by tyronx** [comment]
Thanks that's very helpful. Just one more last favor, does the following shell command opens up a browser for you?

`xdg-open https://www.google.com/`

**Comment by fallerOfFalls** [comment]
Yup it does!

**Comment by tyronx** [comment]
Fixed in v1.12
